### DISCUSSION TITLE
Nuxtr downgrades devtools? (edit: no it does not)

### DISCUSSION THREAD

**Discussion by DavidDeSloovere** [nuxtrdev/nuxtr-vscode#40] [Questions]
It looks like Nuxtr downgrades the devtools to 0.3.1, even though newer version is installed.
Is this possible? 
(Nuxtr v0.1.11)

**Comment by adhamfarrag** [comment]
@DavidDeSloovere shouldn't happen. Depending on your package manager, Nuxtr uses the output from commands npm/pnpm outdated --json and depending on that we show outdated packages. 

Feel free to open an issue and happy to investigate it further.

**Reply by DavidDeSloovere** [reply]
It's so weird. After over an hour I finally got the latest devtools, after removing nuxtr extension, but I can't reproduce it. So thanks for confirming that it should not be nuxtr.

**Reply by adhamfarrag** [reply]
It's weird yeah. This is why I mentioned the method. If you are able somehow to reproduce it once using Nuxtr and then using the terminal, results should match snd the issue here shouldn't be Nuxtr.

**Reply by DavidDeSloovere** [reply]
Finally found it. So definitely not Nuxtr's fault. 😉

This uses new version.
```ts
export default defineNuxtConfig({
  modules: [
    '@nuxt/devtools',
  ]
})
```

This uses older version. 🤯
```ts
export default defineNuxtConfig({
  devtools: { enabled: true }
})
```

Thanks for your time.

**Reply by adhamfarrag** [reply]
Interesting.. when you replaced the configuration it updated properly ?

**Reply by DavidDeSloovere** [reply]
Yes, explicitly adding the module and restarting uses the latest version. It's kind of in the [devtools docs](https://devtools.nuxtjs.org/guide), but it wasn't clear enough for me.

**Reply by adhamfarrag** [reply]
Sorry this is important. Couple questions :)

1. Did you add Nuxt Devtools to your project when it was created through the CLI's question?

I believe the answer is no.. but still doesn't make sense to have Nuxt Devtools stuck at a specific version. 

2. Can you check if Nuxt Devtools is globally installed? 

2. What was the behavior before coming to this conclusion?

**Reply by DavidDeSloovere** [reply]
Sorry. I forgot this reply and now I also forgot what I did then.
I don't have nuxt devtools installed globally.

**Reply by DavidDeSloovere** [reply]
Finally found the solution.
Had to remove lines from the `.nuxtrc` file in my profile dir.
Found it here: https://github.com/nuxt/devtools/issues/42#issuecomment-1437014546
